README.md

```diff
@@ -1,13 +1,13 @@
-Before compiling make sure to create a credentials file named "credentials.py" with variables "clientID" and "clientSecret" and making sure to not commit these confidential keys to the public repository
 
+This project is still in development. Its goal is to fetch and filter relevent NOTAMs for your flight. The data is gathered from the FAA's NOTAM API. This app is built using two frameworks, Flask and Bootstrap.

```

Comment:
It's goal is to fetch and filter relevant NOTAMs for your flight from an easy-to-use web application.